src/java/org/apache/cassandra/index/internal/CassandraIndexSearcher.java

```diff
@@ -114,8 +145,7 @@ private ClusteringIndexFilter makeIndexFilter(ReadCommand command)
         }
         else
         {
-
-            DataRange dataRange = ((PartitionRangeReadCommand)command).dataRange();
+            DataRange dataRange = command.dataRange();
```

Comment:
I could be convinced to leave this cast here almost as an assertion...but not sure it matters either way.